CONTRIBUTING.md

```diff
@@ -301,37 +243,83 @@ This is optional but encouraged.
 Good commit messages explain what changed AND why you changed it.
 See [I've written a clear changelist description](https://github.com/camunda/camunda/wiki/Pull-Requests-and-Code-Reviews#ive-written-a-clear-changelist-description).
 
-## Contributor License Agreement
 
-You will be asked to sign our Contributor License Agreement when you open a Pull Request. We are not
-asking you to assign copyright to us but to give us the right to distribute
-your code without restriction. We ask this of all contributors to
-assure our users of the origin and continuing existence of the code. You only
-need to sign the CLA once.
+## Build Camunda from source
 
-## Licenses
+Camunda is a multi-module Maven project. To **quickly** build all components,
+run the command: `mvn clean install -Dquickly` in the root folder.
 
-Zeebe, Operate, and Tasklist source files are made available under the
-[Camunda License Version 1.0](/licenses/CAMUNDA-LICENSE-1.0.txt) except for the parts listed
-below, which are made available under the [Apache License, Version
-2.0](/licenses/APACHE-2.0.txt).  See individual source files for details.
```

Comment:
❓ Why was this deleted?

Reply:
@korthout -- this is also listed in the README, so I thought it'd be easier to keep it up to date in one place vs. two. Open to changing that if you think it should also be listed here.

Reply:
@korthout, this is also listed in the README, so I thought it'd be easier to keep it up to date in one place vs. two. Open to changing that if you think the licenses should also be listed here.